lib/rspec/mocks/configuration.rb

```diff
@@ -80,13 +80,23 @@ def add_stub_and_should_receive_to(*modules)
       #
       def syntax=(*values)
         syntaxes = values.flatten
+        if self.class.warn_about_syntax?
```

Comment:
We call `configuration.reset_syntaxes_to_default`, and don't want a deprecation message to be emitted at that moment.

Reply:
👍